.github/CODEOWNERS

```diff
@@ -145,3 +145,19 @@ sycl/include/sycl/ext/oneapi/bindless* @intel/bindless-images-reviewers
 sycl/source/detail/bindless* @intel/bindless-images-reviewers
 sycl/plugins/unified_runtime/ur/adapters/**/image.* @intel/bindless-images-reviewers
 sycl/test-e2e/bindless_images/ @intel/bindless-images-reviewers
+
+# sycl e2e tests
+sycl/test-e2e/ @intel/llvm-reviewers-runtime
+sycl/test-e2e/GroupAlgorithm/ @Pennycook @intel/llvm-reviewers-runtime
+sycl/test-e2e/GroupLocalMemory/ @Pennycook @intel/llvm-reviewers-runtime
+sycl/test-e2e/SubGroup/ @Pennycook @intel/llvm-reviewers-runtime
+sycl/test-e2e/SubGroupMask/ @Pennycook @intel/llvm-reviewers-runtime
+sycl/test-e2e/Plugin/*level-zero* @intel/dpcpp-l0-pi-reviewers
+sycl/test-e2e/ESIMD/ @intel/dpcpp-esimd-reviewers
+sycl/test-e2e/InvokeSimd/ @intel/dpcpp-esimd-reviewers
+sycl/test-e2e/BFloat16/ @intel/dpcpp-tools-reviewers
+sycl/test-e2e/AOT/ @intel/dpcpp-tools-reviewers
+sycl/test-e2e/DeviceCodeSplit/ @intel/dpcpp-tools-reviewers
+sycl/test-e2e/SeparateCompile/ @intel/dpcpp-tools-reviewers
+sycl/test-e2e/Printf/ @intel/dpcpp-tools-reviewers
+sycl/test-e2e/SpecConstants/ @intel/dpcpp-tools-reviewers
```

Comment:
Since these might also have part runtime ownership, should it maybe include both?

Reply:
I would agree with this and would add RT reviewers also to `BFloat16` and `Printf` tests. I don't think that either of them necessary has tools-only implementation contrary to `DeviceCodeSplit`, for example.

Reply:
I would prefer a single primary codeowner and then that codeowner can request reviews/help from others if needed. I am ok with multiple owners, but I do feel the first one listed must be the primary owner.
For SpecConstants, runtime team will have primary ownership.

Reply:
I updated the lines per @AlexeySachkov  comment to include second owner. If and when this becomes to confusing and we have to go to single owner per @asudarsa  we can update then.